packages/catalog-process/src/config/config.ts

```diff
@@ -16,8 +16,18 @@ const CatalogProcessConfig = CommonHTTPServiceConfig.and(ReadModelDbConfig)
       .object({
         ESERVICE_DOCUMENTS_PATH: z.string(),
         PRODUCER_ALLOWED_ORIGINS: z.string(),
+        FEATURE_FLAG_SIGNALHUB_WHITELIST: z
+          .enum(["true", "false"])
+          .transform((value) => value === "true"),
```

Comment:
```suggestion
        FEATURE_FLAG_SIGNALHUB_WHITELIST: z
          .enum(["true", "false"])
          .default("false")
          .transform((value) => value === "true"),
```
As suggested by @galales we should set a default in case the env variable is not set and I also agree to define dedicated featureFlags configs, you could do it like [this](https://github.com/pagopa/interop-be-monorepo/blob/4ec2d1ca31a5887c89be31e015a87e2e035d65da/packages/commons/src/config/sessionTokenGenerationConfig.ts#L3) maybe by creating a new file featureFlags that we will use to define all the flags' configs

Reply:
Moved `FEATURE_FLAG_SIGNALHUB_WHITELIST` on dedicated file, however I let `SIGNALHUB_WHITELIST` var on catalog-process config file. I think that whitelist is closely related to catalog-process. Let me know what you think about it.

Reply:
> Moved `FEATURE_FLAG_SIGNALHUB_WHITELIST` on dedicated file, however I let `SIGNALHUB_WHITELIST` var on catalog-process config file. I think that whitelist is closely related to catalog-process. Let me know what you think about it.

I think that it should be included inside the dedicated config since that variable is strictly related to  `FEATURE_FLAG_SIGNALHUB_WHITELIST`, so these should be together IMHO. 

Even if, for now, this config is used only in the catalog-process having all the featureFlags configs in commons would be easier for future development

Reply:
you've convinced me! moved var into feature flag config file 👍